server/views/partials/layout.njk

```diff
@@ -39,7 +44,7 @@
     tag: {
       text: "feedback"
     },
-    html: '<a class="govuk-link" href="https://eu.surveymonkey.com/r/369L5GL">Give us your feedback</a> to help us improve this service.',
+    html: '<a class="govuk-link" data-ga-action="click" data-ga-category="Feedback" data-ga-label="Survey Monkey 369L5GL" data-ga-value="1" href="https://eu.surveymonkey.com/r/369L5GL">Give us your feedback</a> to help us improve this service.',
```

Comment:
I have added the javascript event handler to look for these in any elements on the page and automatically send an event when clicked. Hopefully this helps cover a lot of use cases for now and makes it easy to add events as you need.